node-tests/fixtures/violations.json

```diff
@@ -0,0 +1,3325 @@
+{
```

Comment:
I think we need to tweak the format into something akin to:

```js
[
  {
    testName: 'asdf',
    results: [
      {
        helperName: 'click',
        stack: '',
        label: 'end',
        axeResult: WhateverFromAxeHere
      }    
    ]
  }
]
```

Reply:
Ya I can explore doing something like what we did for test isolation validation in the capturing of the stack. I think we can use the `testMetaData` feature we added to `@ember/test-helpers` to get the helpers that were invoked, the last being the one that triggered the audit.

Reply:
Ya, if we can use the test meta data stuff we might even avoid needing to use QUnit itself to grab the info?

Reply:
So `results` wouldn't be an array. It'd be an `AxeResults` instance, which is an object that looks like [this](https://raw.githubusercontent.com/scalvert/ember-a11y-testing/reporter-middleware/node-tests/fixtures/violations.json), as you referenced. That's the raw `AxeResults` value we get back from an individual run. This would just be an object.

I'd propose we do the following:

```json
{
  testName: 'asdf',
  result: 
  {
    helperName: 'click',
    stack: '',
    axeResult: WhateverFromAxeHere
  }    
}
```

I'm not sure what value the label would provide, so I've removed it. Also, if we wanted that info, we'd need to thread it through the hooks invocation, but that's not guaranteed to be present, as you can invoke `a11yAudit` on its own with this custom middleware.